ocs_ci/helpers/multiple_device_classes.py

```diff
@@ -0,0 +1,163 @@
+import logging
```

Comment:
this file could be used for anything related to device classes and not only for the multiple aspect of it.
Hence, suggest renaming the file to device_class.py and better to place it under ocs_ci/ocs

Reply:
Yes, I will change the name.